sdks/python/apache_beam/io/restriction_trackers.py

```diff
@@ -159,7 +162,8 @@ def try_split(self, fraction_of_remainder):
             cur + int(max(1, (self._range.stop - cur) * fraction_of_remainder)))
         if split_point < self._range.stop:
           prev_stop, self._range.stop = self._range.stop, split_point
-          return (self._range.start, split_point), (split_point, prev_stop)
+          return (OffsetRange(self._range.start, split_point),
```

Comment:
I wonder if a `OffsetRange.split(split_pos)` method would make sense here?

Reply:
Yes, I agree. There is a `OffsetRange.split(bundle_size)` to perform initial split already. If we want to provide splitting into 2 pieces based on given pos, we need to have a different naming, like 'split_at'?